### PR TITLE
Incorporate FAQ doc into existing website FAQ and rework

### PR DESCRIPTION
**Pull request by kenfinnigan**

### CODE REVIEW

**Review by aalmiray — APPROVED** [review]

File: FAQ.md [hunk 1]
```diff
@@ -4,52 +4,207 @@ status: draft
...
+sustainability of open source projects.
+By offering a neutral home for succession planning, a minimal governance model,
+and fiscal sponsorship,
+it aims to ensure these projects remain accessible and continue to thrive
+for future generations of developers and businesses.
+
+## What led to the creation of the Commonhaus Foundation?
+
+The Commonhaus Foundation was inspired by recognizing specific unmet needs within the open source community.
+Especially for self-governing and code-centric open source projects.
```
Comment (gavinking) [inline FAQ.md:19]: This isn't a sentence.

Comment (kenfinnigan) [reply FAQ.md]: I've tweaked it. Is it better now?

And thanks for the prompt review!

File: FAQ.md [hunk 2]
```diff
@@ -4,52 +4,207 @@ status: draft
...
+allowing them to maintain their current infrastructure and processes upon joining.
+We offer flexibility in licensing,
+supporting a broader array of OSI-approved licenses beyond the Apache-2.0.
 
 ## Why not the Eclipse Foundation?
 
-The Eclipse Foundation (EF) has been around almost as long as the ASF, and is home to many open source projects as well.
-In addition to open source projects, the EF oversees specification projects, particularly in relation to JakartaEE.
+The Eclipse Foundation (EF) has been around almost as long as the ASF,
+and is home to many open source projects as well.
```
Comment (gavinking) [inline FAQ.md:103]:
```suggestion
and is also home to many open source projects.
```

File: FAQ.md [hunk 3]
```diff
@@ -4,52 +4,207 @@ status: draft
...
+In addition to open source projects, the EF oversees specification projects,
+particularly in relation to JakartaEE.
```
Comment (gavinking) [inline FAQ.md:105]:
```suggestion
particularly in relation to Jakarta EE.
```

File: FAQ.md [hunk 4]
```diff
@@ -4,52 +4,207 @@ status: draft
...
-At Commonhaus Foundation, we prioritize the project's brand and its community, offering a collaborative environment without mandating changes to project names or aligning strictly with our branding. While supportive of projects that implement or are based on standards, the Commonhaus Foundation does not direct its focus towards the oversight of standards or specifications development, leaving projects free to navigate these aspects as they see fit.
-
-## Facilitating collaborative progress
-
-Both the Apache and Eclipse Foundations offer vital structural supports where competing interests need to be navigated with care.
-
-At Commonhaus, we see our role as facilitators of a different sort of consensus-building—one that encourages projects to agree on how to collaborate and move forward together, without requiring an externally-imposed structure.
+At Commonhaus Foundation, we prioritize the project's brand and its community,
+offering a collaborative environment without mandating changes to project names
+or aligning strictly with our branding.
```
Comment (gavinking) [inline FAQ.md:111]:
```suggestion
or strict alignment with foundation branding.
```

File: FAQ.md (hunk 1 above)
Comment (gavinking) [reply FAQ.md]: > Is it better now?

Yes.

**Comment by kenfinnigan** [comment]
Thanks for all the reviews @gavinking, much appreciated!

**Pull request merged**